Sources/Runner/main.swift

```diff
@@ -1,134 +1,8 @@
 import Foundation
 
-// Pull in the JSON from Danger JS
-let standardInput = FileHandle.standardInput
-let input = standardInput.readDataToEndOfFile()
-
-// Set up some example paths for us to work with
-let path = NSTemporaryDirectory()
-let dslJSONPath = path + "danger-dsl.json"
-let dangerResponsePath = path + "danger-response.json"
-
-let fileManager = FileManager.default
-
-// Create the DSL JSON file for the the runner to read from
-if !fileManager.createFile(atPath: dslJSONPath, contents: input, attributes: nil) {
-    print("Could not create a temporary file for the Dangerfile DSL at: \(dslJSONPath)")
-    exit(0)
-}
-
-// Finds first occurrence of supported path
-let supportedPaths = ["Dangerfile.swift", "danger/Dangerfile.swift", "Danger/Dangerfile.swift"]
-let resolvedPath = supportedPaths.first { fileManager.fileExists(atPath: $0) }
-
-// Exit if a dangerfile was not found at any supported path
-guard let dangerfilePath = resolvedPath else {
-    print("Could not find a Dangerfile")
-    print("Please use a supported path: \(supportedPaths)")
-    exit(0)
-}
-
-// Is this a dev build: e.g. running inside a cloned danger/danger-swift
-let libraryFolders = [
-    ".build/debug", // Working in Xcode / CLI
-    ".build/x86_64-unknown-linux/debug", // Danger Swift's CI
-    ".build/release", // Testing prod
-    "/usr/local/lib/danger" // Homebrew installs lib stuff to here
-]
-
-// for testing prod, do a `make install` first
-//let libraryFolders = ["/usr/local/lib/danger"]
-
-// Was danger-swift installed via marathon?
-// e.g "~/.marathon/Scripts/Temp/https:--github.com-danger-danger-swift.git/clone/.build/release"
-let marathonDangerDLDir = NSHomeDirectory() + "/.marathon/Scripts/Temp/"
-let marathonScripts = try? fileManager.contentsOfDirectory(atPath: marathonDangerDLDir)
-var depManagerDangerLibPaths: [String] = []
-
-if marathonScripts != nil {
-    // TODO: Support running from a fork?
-    let dangerSwiftPath = marathonScripts!.first { return $0.contains("danger-swift") }
-    if dangerSwiftPath != nil {
-        let path = marathonDangerDLDir + dangerSwiftPath! + "/clone/.build/release"
-        depManagerDangerLibPaths.append(path)
-    }
-}
-
-// Check and find where we can link to libDanger from
-let libDanger = "libDanger.dylib"
-let libPaths = libraryFolders + depManagerDangerLibPaths
-
-
-func isTheDangerLibPath(path: String) -> Bool {
-    return fileManager.fileExists(atPath: path + "/libDanger.dylib")  || // OSX
-           fileManager.fileExists(atPath: path + "/libDanger.so")        // Linux
+let cliLength = CommandLine.arguments.count
+if cliLength > 0 && "edit" == CommandLine.arguments[0] {
```

Comment:
I thought `CommandLine` was deprecated. Should use `ProcessInfo.processInfo.arguments` instead. Also, consider `arguments[0]` is the executable.